crawler.py

```diff
@@ -203,7 +210,13 @@ def run_all(driver):
 
     # Webdriver uses Firefox Binaries from downloaded cov build
     driver = webdriver.Firefox(firefox_binary='tools/firefox/firefox-bin')
+
+    # All steps are stored in data folder
```

Comment:
Instead of storing data in the `data` folder and then copying it in the other folder, we can pass it to `run_all` and make it store data directly in the right folder.